api/models.py

```diff
@@ -632,6 +644,43 @@ def __str__(self):
         return self.name
 
 
+class EventFeaturedDocument(models.Model):
+    event = models.ForeignKey(
+        Event,
+        verbose_name=_('event'),
+        on_delete=models.CASCADE,
+        related_name="featured_documents",
+        related_query_name="featured_document",
+    )
+    title = models.CharField(verbose_name=_('title'), max_length=200)
+    description = models.TextField(verbose_name=_('description'))
+    thumbnail = models.ImageField(
+        verbose_name=_('thumbnail'), upload_to='event-featured-documents/thumbnail/',
+        help_text=_('Please maintain aspect ratio (3:4) of image while uploading'),
```

Comment:
minor: is that aspect ratio right? a more standard aspect ratio is 4:3 - should we just clarify width vs height to make this clearer?

Reply:
@frozenhelium is suggesting `96x128`. Is this okay?

Reply:
If we do want something like 96x128, then 3:4 should be fine -- are we enforcing some sort of max upload size or so? I'm guessing we are not doing any thumbnailing ourselves so relying on the user to resize images here? Then it might make sense to give the suggestion in pixel values + maybe enforce a max size somewhere? Users will inevitably try and upload a 30MB image file and we should be prepared :-)

Reply:
Since the image is in portrait format the aspect ratio would be 3:4
@thenav56 may be we can make following the help text?
`For the better result please make sure the thumbnail has resolution close to 96x128`
Does that look good @batpad @tovari

Reply:
To make this a bit clearer for users, can we say:

`Image should be portrait (3:4 aspect ratio) and scaled down to as close to 96x128 as the image size`

@thenav56 if not for the current PR, can we ticket enforcing some sort of sane max image size on uploads?

Reply:
Update the help text and create a new issue for adding validation https://github.com/IFRCGo/go-api/issues/1121.